build/configs/sidk_s5jt200/.debugSpec.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2017 Samsung Electronics All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ either express or implied. See the License for the specific
+ language governing permissions and limitations under the License.
+-->
+<!--
+    predefined macro values:
+    * PROJECT_PATH - current project path
+    * BOARD - selected board name
+    * SCRIPT_PATH - selected script path
+    * SCRIPT_FILE - selected script file
+-->
+<debugs>
+    <debug name="openocd debug" type="openocd">
+        <!-- path attribute value must be relative path from project root -->
+        <!-- usage : openocd -f debug_all.cfg -->
+        <options name="scriptfile" type="file" path="${PROJECT_PATH}/build/configs/${BOARD}/tools/openocd" filter="*.cfg"/>
```

Comment:
Is it *.cfg? not s5jt200_attach_gdb.cfg?

Reply:
This code means all cfg files in ${PROJECT_PATH}/build/configs/${BOARD}/tools/openocd.
So user can select one of them.

Reply:
For debug, there is only one cfg file. Why do you think that we can use all of cfg for debug?